src/katello/agent/pulp/test.py

```diff
@@ -0,0 +1,66 @@
+from libdnf import *
```

Comment:
I guess this file should be under ./test/ somewhere :)

... which leads me to testing! It is definitely concerning that there are hardly any here especially because it could create a kind of behavior where devs are less enthusiastic about writing new tests if there are none/few to begin with. Are there tests for the libs from which this code was derived that could be adjusted to work here or would testing this be all new code?

Reply:
The `AttributeError: 'LibYum' object has no attribute 'preconf'` has me puzzled at the moment.  The `__init__()` code is almost exactly the same as it has been in pulp.  Will dig into it.

As for the tests, we *could* cannibalize the tests from pulp but they mock everything and don't catch integration problems which I think poses the greatest risk.  But, might be better than nothing.